### ISSUE TITLE
options=temporalValues creating 500 error

### ISSUE THREAD

**Issue by hlange** [ScorpioBroker/ScorpioBroker#277]
Hello,
when requesting values with options=temporalValues:
`http://192.168.0.6:9090/ngsi-ld/v1/temporal/entities/house2:smartrooms:room1/?attrs=myuniqueuri:temperature&options=temporalValues`

I get an error 500.
```
{
  "type": "http://uri.etsi.org/ngsi-ld/errors/InternalError",
  "title": "Internal error",
  "details": "null"
}
```

**Comment by RupamNEC** [comment]
Hi @hlange,
This request is not valid, timerel, timeAt is required with options.

Thanks & Regards
Rupam Kumari

**Comment by hlange** [comment]
I don't think so.

https://ngsi-ld-tutorials.readthedocs.io/en/latest/short-term-history.html#simplified-temporal-representation-of-an-entity
```
curl -G -X GET 'http://localhost:8080/temporal/entities/urn:ngsi-ld:Animal:cow001' \
  -H 'NGSILD-Tenant: openiot' \
  -H 'Link: <http://context/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
  -d 'lastN=3' \
  -d 'options=temporalValues'
```
This does not include these parameters.

**Comment by RupamNEC** [comment]
Yes, correct we will fix this issue asap.

**Comment by amitrnec** [comment]
Hi,
We have fixed this issue #280

Thanks,
Amit.
